src/rpc/methods/chain.rs

```diff
@@ -869,6 +869,58 @@ quickcheck::quickcheck! {
     }
 }
 
+pub async fn get_parent_receipts<DB: Blockstore + Send + Sync + 'static>(
+    data: Ctx<DB>,
+    message_receipts: Cid,
+) -> Result<Vec<ApiReceipt>> {
+    let store = data.state_manager.blockstore();
+
+    let mut receipts = Vec::new();
+
+    // Try Receipt_v4 first. (Receipt_v4 and Receipt_v3 are identical, use v4 here)
+    if let Ok(amt) = Amt::<fvm_shared4::receipt::Receipt, _>::load(&message_receipts, store)
```

Comment:
It'd make sense to have a `shim`, so that the usage for `fvmX` is not sprawled across the entire codebase. We utilize this pattern quite a bit there. What do you think?

Reply:
Sounds good.

Reply:
Done. And I will refactor some other functions in a subsequent PR.